lib/nerves_hub_link_common/update_manager.ex

```diff
@@ -146,14 +148,13 @@ defmodule NervesHubLinkCommon.UpdateManager do
     {:noreply, state}
   end
 
-  @spec maybe_update_firmware(map(), State.t()) ::
+  @spec maybe_update_firmware(UpdateInfo.t(), State.t()) ::
           State.download_started() | State.download_rescheduled() | State.t()
-  defp maybe_update_firmware(%{update_available: false}, %State{} = state) do
-    # if the `update_available` key is false, bail early. There is no update
-    state
-  end
 
-  defp maybe_update_firmware(_data, %State{status: {:updating, _percent}} = state) do
+  defp maybe_update_firmware(
+         %UpdateInfo{} = _update_info,
+         %State{status: {:updating, _percent}} = state
+       ) do
     # Received an update message from NervesHub, but we're already in progress.
```

Comment:
Not this PR, but we'll need to revisit this comment. For slow updates over LTE, we'll want to stop in progress updates since they may take >1 hour.